app/controllers/tag_controller.rb

```diff
@@ -287,25 +287,33 @@ def create
     node = Node.find nid
     tagnames.each do |tagname|
       # this should all be done in the model:
-
+      tagname = tagname.strip
       if Tag.exists?(tagname, nid)
         @output[:errors] << I18n.t('tag_controller.tag_already_exists')
+
+      elsif tagname.include?(":") && tagname.split(':').length < 2 # cant tag empty power tags
```

Comment:
You mean `Can't add empty power tags` :sweat_smile: ?

Reply:
Hi @gauravano 
What I mean was we can add `with` but not `with:` ? :sweat_smile: 
Is this not supposed to be like this ?

Reply:
I was referring to comment not code :sweat_smile:

Reply:
Ahh!! Forgot about that. I'll fix it.